network/src/protocols/identity.rs

```diff
@@ -1,44 +1,14 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-//! Protocol used to identify key information about a remote
-//!
-//! Currently, the information shared as part of this protocol includes the peer identity and a
-//! list of protocols supported by the peer.
+//! Protocol used to exchange supported protocol information with a remote.
+
 use crate::protocols::wire::handshake::v1::HandshakeMsg;
 use bytes::BytesMut;
 use futures::io::{AsyncRead, AsyncWrite, AsyncWriteExt};
-use libra_types::PeerId;
 use netcore::framing::{read_u16frame, write_u16frame};
 use std::io;
 
-/// The PeerId exchange protocol.
```

Comment:
makes sense, the peer id exchange should happen in noise anyway since we authenticate against it.